impl/pom.xml

```diff
@@ -411,6 +435,16 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-source-plugin</artifactId>
                 <version>3.2.1</version>
+                <configuration>
+                   <excludes>
+                      <exclude>jakarta/**</exclude>
+                      <exclude>com/sun/faces/vendor/GlassFish*</exclude>
+                      <exclude>com/sun/faces/vendor/Jetty*</exclude>
+                      <exclude>com/sun/faces/vendor/Tomcat*</exclude>
```

Comment:
It might be possible to remove the above three excludes here too if these files no longer exist.

Reply:
Fixed